plugins/javadoc/src/main/kotlin/javadoc/JavadocPageCreator.kt

```diff
@@ -148,21 +129,50 @@ open class JavadocPageCreator(
     private fun DFunction.toJavadocFunction(sourceSetData: SourceSetData) = JavadocFunctionNode(
         name = name,
         signature = signatureProvider.signature(this),
-        brief = TextNode(description(sourceSetData), setOf(sourceSetData)),
+        brief = brief(sourceSetData),
         parameters = parameters.map {
             JavadocParameterNode(
                 name = it.name.orEmpty(),
                 type = signatureForProjection(it.type),
-                description = TextNode(it.findNodeInDocumentation<Param>(sourceSetData), setOf(sourceSetData))
+                description = it.brief()
             )
         },
         extras = extra
     )
 
-    private fun Documentable.description(sourceSetData: SourceSetData): String =
-        findNodeInDocumentation<Description>(sourceSetData)
+    private val Documentable.jvmSource
+        get() = sourceSets.filter { it.platform == Platform.jvm }
 
-    private inline fun <reified T : TagWrapper> Documentable.findNodeInDocumentation(sourceSetData: SourceSetData): String =
-        documentation[sourceSetData]?.children?.firstIsInstanceOrNull<T>()?.root?.children?.firstIsInstanceOrNull<Text>()?.body.orEmpty()
+    private val Documentable.mostTopSourceSet
```

Comment:
"mostTopSourceSet" is not conveying enough information about this property.
Naming suggestion: "firstJvmSourceSet"

Reply:
Actually, this is not the "first" one, this is the first for which we have an `actual` value, so it will be the "most top" one

Reply:
-> "mostTopJvmSourceSet"

Reply:
or "-> highestJvmSourceSet"

Reply:
renamed